test/sizzle.v2.1.0.js

```diff
@@ -0,0 +1 @@
+../lib/sizzle.v2.1.0.js
```

Comment:
Suggest naming this link test/sizzle.js (without the version number)... similar to how we do with other lib/ dependencies.